javascript.js

```diff
@@ -11,4 +11,288 @@ document.getElementById('open-menu').onclick = openNav;
+    alt="popup image" width="311" height="220"></img>
+    <div class="popup-right">
+    <p class="p-Project p-popup">${projectsArray[num].popupdescription}</p>
+    <div class="popup-btn-container">
+    <button type="button" class="popup-btn" href=${projectsArray[num].live}>
+    See Live <i class="fas fa-external-link-alt darkicon"></i></button>
+    <button type="button" class="popup-btn btn-ml" href=${projectsArray[num].source}>
+    See Source <i class="fab fa-github darkicon" title="github icon"></i></button>
+    </div>
+    </div>
+    </div>
+  </div>`;
+
+  document.querySelector('html').style.overflow = 'hidden';
+  const startpopup = document.getElementById('popup');
+  function erase() {
+    while (startpopup.firstChild) {
+      startpopup.removeChild(startpopup.lastChild);
+    }
+    document.querySelector('html').style.overflow = 'scroll';
+  }
+
+  document.getElementById('close-popup').addEventListener('click', () => {
+    erase();
+  });
+}
+document.getElementById('recentWork').innerHTML = `
+<img class="img-works" width="327" height="237" alt="works image" src=${projectsArray[6].image}>
+<div class="works-info">
+<h3 class="works-title-2">${projectsArray[6].title}</h3>
+<p class="p-work">${projectsArray[6].paragraph}</p>
+<ul>
+  <li class="li2">${projectsArray[6].tecnologies[0]}</li>
+  <li class="li2">${projectsArray[6].tecnologies[1]}</li>
+  <li class="li2">${projectsArray[6].tecnologies[2]}</li>
+  <li class="li2">${projectsArray[6].tecnologies[3]}</li>
+</ul>
+<button type="button" id="btn-project-work" class="see-project-btn">See Project</button>
+</div>`;
+
+document.getElementById('projects').innerHTML = `
+<section class="Project-1" id="project-1"> 
+ <h3 class="title-p">${projectsArray[0].title}</h3>
+<p class="p-Project">${projectsArray[0].paragraph}
+</p>
+<ul class="ul-p">
+  <li class="li-p">${projectsArray[0].tecnologies[0]}</li>
+  <li class="li-p">${projectsArray[0].tecnologies[1]}</li>
+  <li class="li-p">${projectsArray[0].tecnologies[2]}</li>
+</ul>
+<button type="button" id="btn-project-1" class="btn-p">See Project</button>
+</section>
+
+ 
+<section class="Project-2" id="project-2"> 
+ <h3 class="title-p">${projectsArray[1].title}</h3>
+<p class="p-Project">${projectsArray[1].paragraph}
+</p>
+<ul class="ul-p">
+  <li class="li-p">${projectsArray[1].tecnologies[0]}</li>
+  <li class="li-p">${projectsArray[1].tecnologies[1]}</li>
+  <li class="li-p">${projectsArray[1].tecnologies[2]}</li>
+</ul>
+<button type="button" id="btn-project-2" class="btn-p">See Project</button>
+</section>
+
+ 
+<section class="Project-3" id="project-3"> 
+ <h3 class="title-p">${projectsArray[2].title}</h3>
+<p class="p-Project">${projectsArray[2].paragraph}
+</p>
+<ul class="ul-p">
+  <li class="li-p">${projectsArray[2].tecnologies[0]}</li>
+  <li class="li-p">${projectsArray[2].tecnologies[1]}</li>
+  <li class="li-p">${projectsArray[2].tecnologies[2]}</li>
+</ul>
```

Comment:
- [ ] [OPTIONAL] See my comment above about hard-coded index values and make the same change here to help keep your code dry